server/routerlicious/packages/services-core/src/configuration.ts

```diff
@@ -145,6 +145,11 @@ export interface IScribeServerConfiguration {
 
 	// Enables scrubbing user data from protocol state quorum in global checkpoints
 	scrubUserDataInGlobalCheckpoints: boolean;
+
+	// Limits the number of service summary versions to track as "valid parent summaries"
+	// since the last client summary. If the list grows beyond this limit, the oldest
+	// service summary version is removed.
+	maxTrackedServiceSummaryVersionsSinceLastClientSummary: number;
```

Comment:
let's further make this configurable?

Reply:
In the "Reviewer Guidance" section of the description, I specifically called this out:

> The configuration is not bubbled up to service level configs yet. I don't think it's necessary.

I think we can come back to make it configurable if we really need to